upi/azure/arm/setup-manifests.py

```diff
@@ -0,0 +1,41 @@
+import json
+import base64
+import sys
+import os
+from dotmap import DotMap
+import yaml
+from collections import OrderedDict 
+
+resource_group = sys.argv[1]
+
+with open('./gw/manifests/cloud-provider-config.yaml') as file:
```

Comment:
this has a hard coded file name

Reply:
It is relative, and it is where the installer will create the file when running the setup_azarm.sh, the python scripts only work in the context of the setup_azarm.sh. We can add a additional command line argument, and pass the "subdirectly" used, but seems to add complexity without alot of gain.